packages/fuels/src/cli/commands/dev/startFuelCore.ts

```diff
@@ -73,8 +75,8 @@ export const startFuelCore = async (config: FuelsConfig): Promise<FuelCoreNode>
   ].flat();
 
   return new Promise((resolve, reject) => {
-    const pkgRootDir = findPackageRoot();
-    const builtInFuelsCorePath = join(pkgRootDir, 'node_modules', '.bin', 'fuels-core');
+    // This line finds the path to the built-in fuels-core binary
```

Comment:
Should we remove the `findPackageRoot` utility?

Or is it being used elsewhere?

Reply:
It is being used in other files